### PR TITLE
Update the OTel Sdk version

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Updated OTel SDK and core components version to latest prerelease `1.1.0-beta1`
- Updated non-core components version to `1.0.0-rc3`

### CODE REVIEW

File: src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry" Version="1.0.1" />
+    <PackageReference Include="OpenTelemetry" Version="1.1.0-beta1" />
```
Comment (cijothomas) [inline src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj:9]: @srprash Please confirm if you are okay to move to next version.

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj (hunk 1 above)
Comment (cijothomas) [reply src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj:9]: @srprash Hi. Ping to see if you want this reverted.

Comment (srprash) [reply src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj:9]: Hi Cijo. Really sorry that I missed this for such a long time. 
Yes, I'm okay to update the OTel version. Will be planning the release later though.

**Review by CodeBlanch — APPROVED** [review]
LGTM

**Pull request merged**